app/javascript/packs/admin.js

```diff
@@ -15,8 +14,3 @@ Rails.start();
 const application = Application.start();
 const context = require.context('admin/controllers', true, /.js$/);
 application.load(definitionsFromContext(context));
-
-initializeDropdown({
-  triggerElementId: 'options-dropdown-trigger',
-  dropdownContentId: 'options-dropdown',
-});
```

Comment:
I removed this from all admin pages because it's currently only used on the edit actions for a user.